docs/05-technical-reference/05-configuration-parameters/smsh-01-istio-chart.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Istio chart
+---
+
+To configure the Istio chart, override the default values of its [`values.yaml`](https://github.com/kyma-project/kyma/blob/main/resources/istio-configuration/values.yaml) file. This document describes parameters that you can configure.
```

Comment:
The link doesn't work.

Reply:
Now the folder's name is `istio` but it will be changed to `istio-configuration` probably this week.